spec/writers/sparql_writer_spec.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+
+require 'traject'
+require 'rialto/etl/writers/sparql_writer'
+
+RSpec.describe Rialto::Etl::Writers::SparqlWriter do
+  subject(:writer) do
+    described_class.new(
+      'sparql_writer.update_url': 'http://127.0.0.1:9999/sparql',
```

Comment:
Are you writing the args this way because Traject wants symbol Hash keys? If so, OK. If not, and if Traject is OK with string Hash keys, it would look a little more idiomatic to use `'foo.bar' => 'http...'` rather than `'foo.bar': 'http...'`.

Reply:
Right, Ruby idioms not python ... Fixing.